task3.py

```diff
@@ -0,0 +1,9 @@
+# Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу между максимальным и минимальным значением дробной части элементов.
+from random import uniform
+array = [round(uniform(1,10), 2) for _ in range(10)]
+maxArray, minArray = round(array[0] % 1, 2) , round(array[1] % 1, 2)
```

Comment:
```suggestion
maxArray, minArray = round(array[0] % 1, 2) , round(array[0] % 1, 2)
```
Если не принять в качестве минимального значения первый элемент, то вы можете его пропустить, так как у вас в 8 строке `elif`